homeassistant/components/knx/__init__.py

```diff
@@ -392,6 +388,38 @@ async def service_event_register_modify(self, call):
                 group_address,
             )
 
+    async def service_exposure_register_modify(self, call):
+        """Service for adding or removing an exposure to KNX bus."""
+        group_address = call.data.get(SERVICE_KNX_ATTR_ADDRESS)
+
+        if call.data.get(SERVICE_KNX_ATTR_REMOVE):
+            try:
+                removed_exposure = self.service_exposures.pop(group_address)
+            except KeyError:
+                _LOGGER.warning(
```

Comment:
Can we raise an exception that will be visible in the UI instead ?

Reply:
How is this done best?

Reply:
Wrap it in HomeAssistantError with a human readable error

Reply:
Thanks! Done.
👍